src/modules/core/components/ClickableHeading/ClickableHeading.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactChild } from 'react';
+
+import Heading from '~core/Heading';
+import NavLink from '~core/NavLink';
+import Icon from '~core/Icon';
+
+import styles from './ClickableHeading.css';
+
+interface Props {
+  children?: ReactChild;
+  linkTo: string;
+}
+
+const ClickableHeading = ({ children, linkTo }: Props) => {
+  return (
+    <div className={styles.heading}>
+      <Heading appearance={{ size: 'smallish', weight: 'bold' }}>
+        <NavLink to={linkTo}>
+          <span className={styles.contents}>
+            {children}
+
```

Comment:
I think you have an extra space here 😉

Reply:
Hmm, it's aligned for me. I wouldn't think a whitespace can make a difference if the parent is using flex. 
<img width="566" alt="image" src="https://user-images.githubusercontent.com/112586815/189313227-aa9ff7f0-af17-4a3b-876e-53953d256ffa.png">
Which browser are you using @ArmandoGraterol?